src/test/resources/application.yaml

```diff
@@ -1,14 +1,18 @@
 app:
   base-url: http://localhost:8080
 
-api:
-  apiKeys:
-  - keyValue: valid-api-key
-    refreshAddress: some-refresh-address
-    dataGroups: [DG1,DG2]
-  - keyValue: other-api-key
-    refreshAddress: other-refresh-address
-    dataGroups: [DG1,DG2]
+tenant:
+  tenants:
+    - id: integration_test_1
+      apiKey: valid-api-key-1
+      refreshAddress: valid-refresh-address-1
+      dataGroups: [ DG1,DG2 ]
+      allowedHosts: ["i.am.allowed.1", "i.am.also.allowed.1"]
+    - id: integration_test_2
+      apiKey: valid-api-key-2
+      refreshAddress: valid-refresh-address-2
+      dataGroups: [ DG4,DG17 ]
+      allowedHosts: ["i.am.allowed.2", "i.am.also.allowed.2"]
```

Comment:
@hannesschaletzky

I am a bit confused with `allowedHosts`. it has a data type of array, which means we allow one or more hosts with the same API key to access our backend (including widgets embedded in multiple hosts). is that a product decision? is there any security risk?